app/controllers/branches_controller.rb

```diff
@@ -23,15 +23,15 @@ def create
       if @branch.save
         @leaf.branch_id = @branch.id
         if @leaf.save
-          flash[:success] = "Branch created!"
+          flash[:success] = 'Branch created!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.